bedrock/settings/base.py

```diff
@@ -535,9 +535,9 @@ def get_app_name(hostname):
     return "bedrock"
 
 
-HOSTNAME = platform.node()
-APP_NAME = get_app_name(HOSTNAME)
+APP_NAME = config("APP_NAME", default="bedrock-noenvironment")
```

Comment:
This should never use the default, so I thought we'd use one that's really clearly a sign of something not being set up as we expect